hail/src/main/scala/is/hail/expr/types/physical/PLocus.scala

```diff
@@ -45,7 +51,7 @@ case class PLocus(rg: RGBase, override val required: Boolean = false) extends Co
         val contig1 = PString.loadString(r1, cOff1)
         val contig2 = PString.loadString(r2, cOff2)
 
-        val c = rg.compare(contig1, contig2)
+        val c = rgBc.value.compare(contig1, contig2)
```

Comment:
Why not use the `rg` defined above which computes the same thing?

Reply:
I was trying to avoid capturing the PLocus type itself in the ordering, but this doesn't work either.  I changed it to use `localRGBc`.  Good catch.